src/algorithms/pid/Tools.h

```diff
@@ -30,11 +30,11 @@ namespace Tools {
   // -------------------------------------------------------------------------------------
   // Radiator IDs
 
-  static std::unordered_map<int, std::string> GetRadiatorIDs() {
+  inline std::unordered_map<int, std::string> GetRadiatorIDs() {
```

Comment:
`inline` keyword doesn't convey anything anymore, we don't need it.

Reply:
You tried this earlier today and it failed. https://github.com/eic/EICrecon/actions/runs/14823758425/job/41614185550?pr=1826#step:7:560

Reply:
Oh, I now remember, such issues can reveal (under clang, if I remember correctly). Let's leave it in then.